debian/rules

```diff
@@ -143,6 +143,16 @@ endif
 	ln -s libmariadb.so.3 $(TMP)/usr/lib/$(DEB_HOST_MULTIARCH)/libmysqlclient.so.19
 	ln -s libmariadb.so.3 $(TMP)/usr/lib/$(DEB_HOST_MULTIARCH)/libmysqlclient.so.20
 
+	# Install libmariadbclient.a compatibility link
+	ln -s libmariadb.a $(TMP)/usr/lib/$(DEB_HOST_MULTIARCH)/libmariadbclient.a
+
+	# Symlink test plugin that is used by byth server and client and thus need to
+	# load from the libmariadb path as well
+	ln -s ../../../mysql/plugin/auth_test_plugin.so $(TMP)/usr/lib/$(DEB_HOST_MULTIARCH)/libmariadb3/plugin/auth_test_plugin.so
+	# Move test plugins that are only needed by the client to the libmariadb path
+	mv -v $(TMP)/usr/lib/mysql/plugin/qa_auth_interface.so $(TMP)/usr/lib/$(DEB_HOST_MULTIARCH)/libmariadb3/plugin/
+	mv -v $(TMP)/usr/lib/mysql/plugin/qa_auth_client.so $(TMP)/usr/lib/$(DEB_HOST_MULTIARCH)/libmariadb3/plugin/
```

Comment:
It'd be cleaner to install plugins directly into the correct location, instead of moving them around in rules.

Reply:
This layout only applies for -DDEB, so it does not hurt that it is managed in the debian/rules file. But I do agree that doing this directly in mysql-test/CMakeLists.txt would make sense, however I don't have enough CMake skills to do it there myself.

Reply:
For qa_auth_client: #1497
qa_auth_interface.so is another dual client/server shared library.

Reply:
@cvicentiu Looked into writing this in CMake, but estimated it would take too much time and not sensible in this round, but can be done later.